test/eredis_tests.erl

```diff
@@ -3,7 +3,7 @@
 -include_lib("eunit/include/eunit.hrl").
 -include("eredis.hrl").
 
--define(assertReceive(Pattern),
+-define(?ASSERT_RECEIVE(Pattern),
```

Comment:
^ Unwarranted question mark.

Reply:
(I'd rather keep this macro named in the same fashion as stdlib's assertion macros, but I suppose there's no right answer to this.)

Reply:
Perhaps you fixed it later? Please resolve if that's the case. I'm sorry, I don't think this commit-by-commit approach to reviews works very well for me...

Reply:
> (I'd rather keep this macro named in the same fashion as stdlib's assertion macros, but I suppose there's no right answer to this.)

There's no correct answer for this. It was either "opening an `elvis` exception for everything else" or "fix this". I preferred to "fix this"... but worry not, a very-near-future `elvis` version will allow for specific macros to be ignored.

Reply:
You may Resolve conversation if the reply is satisfactory or if you want not to continue discussing this subject (in this scope only).

Reply:
Good enough for me.